Source/TextToSpeechV1/Models/CustomVoiceUpdate.swift

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+
```

Comment:
Jazzy, which is the tool we use to create the [documentation](http://watson-developer-cloud.github.io/ios-sdk/), is pretty picky. Unfortunately, if we do not comment a public model or function, the generated documentation will say "Not Documented".

Would you mind adding a quick comment here to describe the `CustomVoiceUpdate` model?